frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segments_view.tsx

```diff
@@ -333,6 +343,9 @@ class SegmentsView extends React.Component<Props, State> {
     if (features().jobsEnabled) {
       this.pollJobData();
     }
+    this.props.segmentGroups.forEach(
+      (group) => (this.state.areSegmentsInGroupVisible[group.groupId] = false),
+    );
```

Comment:
I think you can remove that code. 1) you are directly mutating the state even though it should be updated immutable (see my other comment for that). 2) the default of `false` is not really necessary, since it doesn't matter whether the value is false or whether the key is not in the object.